components/profile-controller/README.md

```diff
@@ -0,0 +1,72 @@
+# Profile CRD
+
+Kubeflow Profile CRD is designed to solve access management within multi-user kubernetes cluster.
+
+Profile access management provide namespace level isolation based on:
+
+- k8s rbac access control
+- Istio rbac access control
+
+**Resources managed by profile CRD:**
+
+Each profile CRD will manage one namespace (with same name as profile CRD), and will have one owner.
+Specifically, each profile CRD will manage following resources:
+
+- Namespace
+- K8s rbac Role & Rolebinding to grant profile owner access to above namespace via k8s API (kubectl)
+- Istio namespace-scoped gateway & rbac permission to grant profile owner access to above namespace via Istio (browser)
```

Comment:
Is the user still able to submit a Profile CR and have the namespace and resources created?
You may want to name all the kinds within the Namespace
- ServiceRole
- ServiceRoleBinding
- Role
- RoleBinding
- etc

Reply:
updated